src/app/handler/user/user.handler.go

```diff
@@ -17,12 +21,24 @@ func NewHandler(service user.Service, validate validator.IDtoValidator) *Handler
 	return &Handler{service, validate}
 }
 
-func (h *Handler) FindOne(c *router.FiberCtx) {
+// FindOne is a function that returns a user by id from database
+// @Summary finds one user
+// @Description Returns the data of user if successful
+// @Param id path string true "user id"
+// @Tags user
+// @Accept json
+// @Produce json
+// @Success 200 {object} dto.User
+// @Failure 400 {object} dto.ResponseBadRequestErr "Invalid request body"
+// @Failure 500 {object} dto.ResponseInternalErr "Internal service error"
+// @Failure 503 {object} dto.ResponseServiceDownErr "Service is down"
+// @Router /v1/users/{id} [get]
+func (h *Handler) FindOne(c router.IContext) {
```

Comment:
Not sure which scenario that this endpoint will be called.

Reply:
in case the frontend needs to display user name in the top of app

Reply:
i see